cmd/kubeops/internal/handler/handler.go

```diff
@@ -55,3 +88,22 @@ func ListReleases(cfg agent.Config, w http.ResponseWriter, req *http.Request, pa
 func ListAllReleases(cfg agent.Config, w http.ResponseWriter, req *http.Request, _ handlerutil.Params) {
 	ListReleases(cfg, w, req, make(map[string]string))
 }
+
+func CreateRelease(cfg agent.Config, w http.ResponseWriter, req *http.Request, params handlerutil.Params) {
+	requireV1Support := false
+	chartDetails, chartMulti, err := handlerutil.ParseAndGetChart(req, cfg.ChartClient, requireV1Support)
+	if err != nil {
+		response.NewErrorResponse(handlerutil.ErrorCode(err), err.Error()).Write(w)
+		return
+	}
+	ch := chartMulti.Helm3Chart
```

Comment:
I get that `requireV1Support` is set to false, but I guess that does not imply that this _is_ in-fact a Helm 3 chart. But here, it seems like we assume that it is a Helm3Chart. It's probably just me not understanding how this works, but I wanted to highlight it in case you've missed a corner-case here.

Reply:
Yeah, it's not an assumption of a helm 3 chart, so much as that we're using the helm3 struct representation of the chart here (which supports both helm3 and helm2 charts), rather than the helm2 struct representation (which supports helm 2 charts only). Perhaps I should have named it `Helm3LoadedChart`... naming is hard :/

Reply:
I think the name is good. The semantics of the parameter in question is such that when it's `true`, `ParseAndGetChart` should fail if it cannot load a v1 (Helm 2) chart, whereas when it's `false`, it may return only a v2 (Helm 3) chart and `nil` for the v1 chart.